src/Cable.php

```diff
@@ -78,6 +78,8 @@ public function defineTabs($options = [])
          ->addStandardTab('Change_Item', $ong, $options)
          ->addStandardTab('Log', $ong, $options);
 
+        // Get parents tabs
+        $ong = array_merge($ong, parent::defineTabs());
```

Comment:
In cases where `$usenotepad` is already active, its tab comes before the `Log` tab.
```suggestion
         ->addStandardTab('Change_Item', $ong, $options)
         ->addStandardTab('Notepad', $ong, $options)
         ->addStandardTab('Log', $ong, $options);
```